projects/demo/src/modules/app/app.providers.ts

```diff
@@ -197,6 +198,26 @@ export const APP_PROVIDERS: Provider[] = [
         useFactory: (baseHref: string) => (tree: UrlTree) =>
             String(tree).replace(baseHref, ``),
     },
+    {
+        provide: TUI_DOC_TYPE_REFERENCE_HANDLER,
+        useValue: (type: string) => {
+            switch (type) {
+                case `any`:
+                case `null`:
+                case `undefined`:
+                case `boolean`:
+                case `string`:
+                case `number`:
+                    return null;
+                case `SafeResourceUrl`:
+                    return `https://angular.dev/api/platform-browser/${type}`;
+                case `PolymorpheusContent`:
```

Comment:
nit: can we handle `T`, `Map<T>`, `PolymorpheusContent<T>` as well? 

`PolymorpheusContent<TuiTreeContext>` refers to an empty search result on this page:

https://taiga-ui--pr6086-splincode-feat-type-6othcr6a.web.app/components/tree/API

Reply:
fixed